holoviews/plotting/bokeh/__init__.py

```diff
@@ -54,7 +54,7 @@
 
                 # Rasters
                 Image: RasterPlot,
-                GridImage: ImagePlot,
+                GridImage: RasterPlot,
```

Comment:
I thought ``GridImage`` was going away...can we not get rid of it here? Or is this for backwards compatibility only?